modules/tinymce/src/core/main/ts/api/Editor.ts

```diff
@@ -886,9 +886,8 @@ class Editor implements EditorObservable {
   /**
    * Returns true/false if the editor is dirty or not. It will get dirty if the user has made modifications to the contents.
    *
-   * The dirty state is automatically set to true if you do modifications to the content in other
-   * words when new undo levels is created or if you undo/redo to update the contents of the editor. It will also be set
-   * to false if you call editor.save().
+   * The dirty state is automatically set to `true` when the editor content is modified by the user since initialization
```

Comment:
I'm not a fan of since here, it feels a little off. "...modified by the user after initialization" perhaps?

Also which English do we use for docs?

Reply:
US English

Reply:
I rephrased to use the active voice so that it would be a little more natural to use "after" rather than "since".

Reply:
Yeah. I dropped the ball on the active/passive bit. Need more sleep.